moped-editor/src/components/FileUpload/FileUploadDialogSingle.js

```diff
@@ -204,6 +205,7 @@ const FileUploadDialogSingle = (props) => {
                 <MenuItem value={3}>Estimates</MenuItem>
                 <MenuItem value={4}>Other</MenuItem>
               </Select>
+              <FormHelperText>Required</FormHelperText>
```

Comment:
during testing I was confused by why i couldnt save, and its because the file type is required. 

I only added it here, but the whole form could probably use some ux love cc: @amenity